README.md

```diff
@@ -150,10 +164,18 @@ module.exports = {
 
 ### `minimizerOptions`
 
-Type: `Object`
+Type:
+
+```ts
+type minimizerOptions = {
+  space?: string | number;
+  replacer?: null | Function;
+};
```

Comment:
As per our schema - https://github.com/webpack-contrib/json-minimizer-webpack-plugin/blob/8a65b2cb7b6c9fb9745872e2b2e36d79c508f0ea/src/options.json#L76-L98
```suggestion
type minimizerOptions = {
  space?: null | string | number;
  replacer?: null | Function | Array<string | number>;
};
```

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#the_replacer_parameter
Im guessing there are more types too? like boolean and Symbol as well?

Reply:
Hmm, looks like we don't support them at the moment.

/cc @alexander-akait

Reply:
Oh, let's add test case for this

Reply:
@snitin315  I've changed the type as per you have mentioned. We shall probably add boolean and symbol along with tests when we add them? Thoughts?